src/__tests__/utils/puzzles.test.ts

```diff
@@ -0,0 +1,18 @@
+import { checkPuzzleAnswer } from '../../utils/puzzles';
+import puzzle from '../mocks/mock_puzzle';
+
+it('should compare the user submitted puzzle answer with the ans key in the puzzle (from the db)', () => {
+  const answer = puzzle.variables.answer;
```

Comment:
what if answer does not appear in variables ?

Reply:
What do you mean? Isn't this the way how Puzzle is set up in the Prisma scheme? Are you suggesting that we shouldn't hard code that (line5) or you meant something else?

Reply:
Because in schema, `variables` is an object, therefore, it cannot guarantee that there's a field named `answer`. You should check if `answer` exist in variables or use `puzzle.variables?.answer`

Reply:
I see!